diff -urN tigervnc-1.1.0/unix/xserver/hw/vnc/Makefile.am tigervnc-1.1.0-patched/unix/xserver/hw/vnc/Makefile.am
--- tigervnc-1.1.0/unix/xserver/hw/vnc/Makefile.am	2012-09-07 02:16:27.673845412 +1100
+++ tigervnc-1.1.0-patched/unix/xserver/hw/vnc/Makefile.am	2012-09-07 02:16:55.888845112 +1100
@@ -62,10 +62,10 @@
 BUILT_SOURCES = $(nodist_Xvnc_SOURCES)
 
 fb.h: $(top_srcdir)/fb/fb.h
-	cat $(top_srcdir)/fb/fb.h | sed -e 's,and,c_and,' -e 's,xor,c_xor,' > $(srcdir)/fb.h
+	cat $(top_srcdir)/fb/fb.h | sed -e 's,and,c_and,g' -e 's,xor,c_xor,g' > $(srcdir)/fb.h
 
 pixman.h: $(includedir)/pixman-1/pixman.h
 	cat $(includedir)/pixman-1/pixman.h | sed 's/xor/c_xor/' > $(srcdir)/pixman.h
 
 fbrop.h: $(top_srcdir)/fb/fbrop.h
-	cat $(top_srcdir)/fb/fbrop.h | sed -e 's,and,c_and,' -e 's,xor,c_xor,' > $(srcdir)/fbrop.h
+	cat $(top_srcdir)/fb/fbrop.h | sed -e 's,and,c_and,g' -e 's,xor,c_xor,g' > $(srcdir)/fbrop.h
