--- tigervnc-1.3.0/unix/xserver/hw/vnc/Makefile.am.0002~	2014-02-18 11:30:58.745803593 +0100
+++ tigervnc-1.3.0/unix/xserver/hw/vnc/Makefile.am	2014-02-18 11:31:35.361802094 +0100
@@ -37,7 +37,7 @@ Xvnc_CPPFLAGS = $(XVNC_CPPFLAGS) -DTIGER
 	-I$(top_srcdir)/include ${XSERVERLIBS_CFLAGS} -I$(includedir)
 
 Xvnc_LDADD = $(XVNC_LIBS) libvnccommon.la $(COMMON_LIBS) \
-	$(XSERVER_LIBS) $(XSERVER_SYS_LIBS) $(XVNC_SYS_LIBS) -lX11
+	$(XSERVER_LIBS) $(XSERVER_SYS_LIBS) $(XVNC_SYS_LIBS) -lX11 -lcrypto
 
 Xvnc_LDFLAGS = $(LD_EXPORT_SYMBOLS_FLAG)
 
