diff -uraN tigervnc-1.14.0/unix/xserver/hw/vnc/Makefile.am omv-tigervnc-1.14.0/unix/xserver/hw/vnc/Makefile.am
--- tigervnc-1.14.0/unix/xserver/hw/vnc/Makefile.am	2024-07-23 10:50:25.000000000 +0200
+++ omv-tigervnc-1.14.0/unix/xserver/hw/vnc/Makefile.am	2024-07-23 18:33:07.547968585 +0200
@@ -1,11 +1,11 @@
 TIGERVNC_SRCDIR=${top_srcdir}/../..
 TIGERVNC_BUILDDIR=${TIGERVNC_SRCDIR}
 
-RFB_LIB=$(TIGERVNC_BUILDDIR)/common/rfb/librfb.la
-RDR_LIB=$(TIGERVNC_BUILDDIR)/common/rdr/librdr.la
-OS_LIB=$(TIGERVNC_BUILDDIR)/common/os/libos.la
-NETWORK_LIB=$(TIGERVNC_BUILDDIR)/common/network/libnetwork.la
-UNIXCOMMON_LIB=$(TIGERVNC_BUILDDIR)/unix/common/libunixcommon.la
+RFB_LIB=$(TIGERVNC_BUILDDIR)/build/common/rfb/librfb.la
+RDR_LIB=$(TIGERVNC_BUILDDIR)/build/common/rdr/librdr.la
+OS_LIB=$(TIGERVNC_BUILDDIR)/build/common/os/libos.la
+NETWORK_LIB=$(TIGERVNC_BUILDDIR)/build/common/network/libnetwork.la
+UNIXCOMMON_LIB=$(TIGERVNC_BUILDDIR)/build/unix/common/libunixcommon.la
 COMMON_LIBS=$(NETWORK_LIB) $(RFB_LIB) $(RDR_LIB) $(OS_LIB) $(UNIXCOMMON_LIB)
 
 AM_CPPFLAGS = \
