diff -up tigervnc-1.16.2/unix/xserver/hw/vnc/Makefile.am.2~ tigervnc-1.16.2/unix/xserver/hw/vnc/Makefile.am
--- tigervnc-1.16.2/unix/xserver/hw/vnc/Makefile.am.2~	2024-01-01 00:00:00.000000000 +0000
+++ tigervnc-1.16.2/unix/xserver/hw/vnc/Makefile.am	2024-01-01 00:00:00.000000000 +0000
@@ -57,7 +57,8 @@ Xvnc_LDADD = \
 	$(LOCAL_LIBS) \
 	$(XSERVER_SYS_LIBS) \
 	$(XVNC_SYS_LIBS) \
-	$(LIBXCVT_LIBS)
+	$(LIBXCVT_LIBS) \
+	-lX11 -lcrypto
 
 Xvnc_LDFLAGS = $(LD_EXPORT_SYMBOLS_FLAG)
 
