diff -Nru a/unix/xserver/hw/vnc/Makefile.am b/unix/xserver/hw/vnc/Makefile.am
--- a/unix/xserver/hw/vnc/Makefile.am	2018-06-13 16:28:57.000000000 +0200
+++ b/unix/xserver/hw/vnc/Makefile.am	2018-07-14 16:34:14.662521811 +0200
@@ -1,12 +1,12 @@
 TIGERVNC_SRCDIR=${top_srcdir}/../..
 TIGERVNC_BUILDDIR=${TIGERVNC_SRCDIR}
 
-RFB_LIB=$(TIGERVNC_BUILDDIR)/common/rfb/librfb.la
-RDR_LIB=$(TIGERVNC_BUILDDIR)/common/rdr/librdr.la
-OS_LIB=$(TIGERVNC_BUILDDIR)/common/os/libos.la
-NETWORK_LIB=$(TIGERVNC_BUILDDIR)/common/network/libnetwork.la
-XREGION_LIB=$(TIGERVNC_BUILDDIR)/common/Xregion/libXregion.la
-UNIXCOMMON_LIB=$(TIGERVNC_BUILDDIR)/unix/common/libunixcommon.la
+RFB_LIB=$(TIGERVNC_BUILDDIR)/build/common/rfb/librfb.la
+RDR_LIB=$(TIGERVNC_BUILDDIR)/build/common/rdr/librdr.la
+OS_LIB=$(TIGERVNC_BUILDDIR)/build/common/os/libos.la
+NETWORK_LIB=$(TIGERVNC_BUILDDIR)/build/common/network/libnetwork.la
+XREGION_LIB=$(TIGERVNC_BUILDDIR)/build/common/Xregion/libXregion.la
+UNIXCOMMON_LIB=$(TIGERVNC_BUILDDIR)/build/unix/common/libunixcommon.la
 COMMON_LIBS=$(NETWORK_LIB) $(RFB_LIB) $(RDR_LIB) $(XREGION_LIB) $(OS_LIB) $(UNIXCOMMON_LIB)
 
 noinst_LTLIBRARIES = libvnccommon.la
